lib/easypost/util.rb

```diff
@@ -151,15 +158,17 @@ def self.validate_webhook(event_body, headers, webhook_secret)
     easypost_hmac_signature = headers['X-Hmac-Signature']
 
     if easypost_hmac_signature.nil?
-      raise EasyPost::Error.new('Webhook received does not contain an HMAC signature.')
+      msg = EasyPost::Constants::WEBHOOK_MISSING_SIGNATURE
+      raise EasyPost::Exceptions::SignatureVerificationError.new(msg)
     end
 
     encoded_webhook_secret = webhook_secret.unicode_normalize(:nfkd).encode('utf-8')
 
     expected_signature = OpenSSL::HMAC.hexdigest('sha256', encoded_webhook_secret, event_body)
     digest = "hmac-sha256-hex=#{expected_signature}"
     unless digest == easypost_hmac_signature
-      raise EasyPost::Error.new('Webhook received did not originate from EasyPost or had a webhook secret mismatch.')
+      msg = EasyPost::Constants::WEBHOOK_SIGNATURE_MISMATCH
+      raise EasyPost::Exceptions::SignatureVerificationError.new(msg)
```

Comment:
suggestion: inline the constant directly to the error, no need to reassign it to a local variable here since it's not used again.

Reply:
Line too long, rubocop complains